docs/docker-workflows.rst

```diff
@@ -21,162 +22,135 @@ In this example, we are going to be using the `DJLServing 0.27.0 with
+    </pre>
 
-    prod_env = rh.env(name="prod_env", env_vars={"HF_TOKEN": "****"})
-    prod_env.to(cluster)
 
 
 .. parsed-literal::
     :class: code-output
 
-    INFO | 2024-08-01 02:19:13.168591 | Port 32300 is already in use. Trying next port.
-    INFO | 2024-08-01 02:19:13.172968 | Running forwarding command: ssh -T -L 32301:localhost:32300 -i ~/.ssh/sky-key -o Port=10022 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ConnectTimeout=30s -o ForwardAgent=yes -o ProxyCommand='ssh -T -L 32301:localhost:32300 -i ~/.ssh/sky-key -o Port=22 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ExitOnForwardFailure=yes -o ServerAliveInterval=5 -o ServerAliveCountMax=3 -o ConnectTimeout=30s -o ForwardAgent=yes -W %h:%p ubuntu@3.142.171.243' root@localhost
-    INFO | 2024-08-01 02:19:16.685047 | Calling prod_env._set_env_vars
+    [0m[32m✓ Cluster launched: diffusers_docker.[0m  [2mView logs at: ~/sky_logs/sky-2024-12-23-13-56-48-619803/provision.log[0m
 
 
 .. parsed-literal::
     :class: code-output
 
-    ----------------
-    [36mdiffusers_docker[0m
-    ----------------
-    [36mprod_env env: Calling method _set_env_vars on module prod_env
-    [0m
+    INFO | 2024-12-23 14:03:39 | runhouse.resources.hardware.launcher_utils:391 | Starting Runhouse server on cluster
+    INFO | 2024-12-23 14:03:39 | runhouse.resources.hardware.cluster:1247 | Restarting Runhouse API server on diffusers_docker.
 
-.. parsed-literal::
-    :class: code-output
+    INFO:     Started server process [2929]
+    INFO:     Waiting for application startup.
+    INFO:     Application startup complete.
+    INFO:     Uvicorn running on http://0.0.0.0:32300 (Press CTRL+C to quit)
 
-    INFO | 2024-08-01 02:19:17.273890 | Time to call prod_env._set_env_vars: 0.59 seconds
-    INFO | 2024-08-01 02:19:17.350932 | Calling prod_env.install
 
 
-.. parsed-literal::
-    :class: code-output
 
-    [36mprod_env env: Calling method install on module prod_env
-    [0m
 
 .. parsed-literal::
     :class: code-output
 
-    INFO | 2024-08-01 02:19:17.929387 | Time to call prod_env.install: 0.58 seconds
+    <runhouse.resources.hardware.on_demand_cluster.OnDemandCluster at 0x127ea7730>
 
 
 
+Sample Function
+---------------
 
-.. parsed-literal::
-    :class: code-output
+The function we’ll be using in our demo is ``is_transformers_available``
+from ``diffusers.utils``. We’ll first show how to use the base version
+of this function, which was installed on the box through the cluster
+setup (e.g. a production setting). Then, we’ll show how if you have a
+locally edited version of the same function/package on the cluster, how
+to propagate up your local changes and run them on the cluster.
```

Comment:
small nit, what do you think about: 
```
Then, we’ll show how
to propagate up your local changes and run them on the cluster, if you have a
locally edited version of the same function/package on the cluster.
```
instead of: 
```
Then, we’ll show how if you have a
locally edited version of the same function/package on the cluster, how
to propagate up your local changes and run them on the cluster.
```

(there double usage of how, right after "if" and then after the comma, feels a bit redundant)